zoom_rooms/indico_zoom_rooms/tasks.py

```diff
@@ -0,0 +1,115 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2024 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from pprint import pformat
+
+import requests
+from celery.schedules import crontab
+
+from indico.core.celery import celery
+from indico.core.db import db
+from indico.core.plugins import current_plugin
+from indico.util.string import strip_control_chars
+
+from indico_zoom_rooms.models import ZoomRoomsAction, ZoomRoomsQueueEntry
+
+
+def _send_request(method: str, zr_id: str, entry_id: str, data: dict | None = None) -> bool:
+    """Send a request to the corresponding HTTP service."""
+    from indico_zoom_rooms.plugin import ZoomRoomsPlugin
+    logger = current_plugin.logger
+
+    path = f'/api/v1/users/{zr_id}/events/{entry_id}'
+    if not (url := ZoomRoomsPlugin.settings.get('service_url')):
+        raise KeyError('service_url is not set!')
+    url = url.rstrip('/') + path
+
+    if not (token := ZoomRoomsPlugin.settings.get('token')):
+        raise KeyError('token is not set!')
+    try:
+        res = requests.request(
+            method,
+            url,
+            json=data,
+            headers={'Authorization': f'Bearer {token}'},
+            timeout=ZoomRoomsPlugin.settings.get('timeout'),
+        )
+        logger.info('%s request to %s finished with status %r and body %r', method, path, res.status_code, res.text)
+        res.raise_for_status()
+        return True
+    except requests.Timeout:
+        logger.warning('Request timed out')
+    except requests.RequestException:
+        logger.exception('%s request failed:\nURL: %s\nData: %s', method, url, pformat(data))
+    except requests.HTTPError:
+        logger.error(
+            'Request unsuccessful:\nURL: %s\nData: %s\nCode: %s\nResponse: %s',
+            url,
+            pformat(data),
+            res.status_code,
+            res.text,
+        )
+
+    return False
+
+
+def put_entry(zr_id: str, entry_id: str, entry: dict) -> bool:
+    """Trigger a PUT request (creation and update)."""
+    return _send_request(
+        'PUT',
+        zr_id,
+        entry_id,
+        {
+            'status': 'BUSY',
+            'start': entry['start_dt'],
+            'end': entry['end_dt'],
+            'subject': strip_control_chars(entry['title']),
+            'body': f'<a href="{entry['url']}">{entry['url']}</a>',
+        },
+    )
+
+
+def delete_entry(zr_id: str, entry_id: str) -> bool:
+    """Trigger a DELETE request."""
+    return _send_request('DELETE', zr_id, entry_id)
+
+
+@celery.periodic_task(run_every=crontab(minute='*/1'), plugin='vc_zoom')
```

Comment:
```suggestion
@celery.periodic_task(run_every=crontab(), plugin='vc_zoom')
```

Or if you want to be explicit, `minute='*'` - but all those values default to `*` anyway...

Reply:
I'd rather leave it as `minute='*'`. Not everyone is aware that the minimum granularity is 1min and being explicit is better in this case.